documentation/rosetta-modelling-component.md

```diff
@@ -662,6 +663,66 @@ E.g.:
 [DayOfWeekEnum->SAT, DayOfWeekEnum->SUN]
 ```
 
+### Data Type and Record Constructor
+
+Data types and records can be constructed using a syntax similar to JSON, i.e.,
+```
+<type name> {
+  <field name 1>: <field value 1>,
+  <field name 2>: <field value 2>,
+  etc
+}
+```
+
+As an example, consider the following data type representing an employee of a company.
+``` Haskell
```

Comment:
Our "theme" for examples throughout the documentation is related to vehicles. I think we should continue to follow the same convention. And accordingly, those examples could be added to the demo model.